modules/css/banner/lib/accessibility.scss

```diff
@@ -8,7 +8,7 @@
 
 [data-whatinput='keyboard'] {
   @include wdc-focusable-banner() {
-    @extend .wdc-banner.wdc-banner-focus;
+    @extend .wdc-banner-focus;
```

Comment:
Whoever thought of making new class names was the right solution is an evil genius.